jplag.endToEndTesting/src/main/java/de/jplag/end_to_end_testing/model/ResultModel.java

```diff
@@ -0,0 +1,48 @@
+package de.jplag.end_to_end_testing.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * The ResultModel contains all information and comparison values that are important for a test and that could be parsed
+ * from the json file.
+ */
+public class ResultModel {
+
+    @JsonProperty("result_similarity")
+    private float resultSimilarity;
+    @JsonProperty("test_identifier")
+    private Integer testIdentifier;
```

Comment:
int or Integer ?

Reply:
Integer because this is handled differently by the serialization using Jackson than if I use it as int. This is mainly about the handling of null or 0 by the parser.

Reply:
Yes. But do we want to allow `null` here?

Reply:
The type has more to do with error susceptibility and where the user then gets the error message. Here the question is whether it should already happen when loading the JavaResult.json or only when comparing the stored and current values.

Reply:
I thought about it again and `int` is the more suitable `type` for this, because otherwise errors in the json file would not be noticed at all but would be reflected in wrong tests.